src/services/useAwsPlaceService.ts

```diff
@@ -83,6 +84,15 @@ const useAwsPlaceService = () => {
 				};
 
 				return await locationClient?.searchPlaceIndexForPosition(params).promise();
+			},
+			getNLPlacesByText: async (Text: string) => {
+				const response = await fetch(`${NLP_BASE_URL}/places/ask?` + new URLSearchParams({ Text: Text }), {
+					method: "GET",
+					headers: {
+						"x-api-key": "lBrENB1FDv5cDfT3cHQSd9qKJuut3hIU4ZdCeeO8"
+					}
+				});
+				return await response.json();
```

Comment:
Why this needs an `await` provided there is an `await` in line 89

Reply:
response.json() returns a promise so I have to wait for the promise to fulfill.